module_change_auto_install/patch.py

```diff
@@ -53,7 +53,7 @@ def _overload_load_manifest(module, mod_path=None):
         # Specific case where a previously available module marked as auto installable
         # is NOT available in the addons path.
         # In that case, avoid to crash when trying to get 'depends' key.
-        return
+        return {}
```

Comment:
Better return super value, dont you think ?
```suggestion
        return res
```